src/main/java/org/kuali/kfs/module/purap/document/dataaccess/PaymentRequestDao.java

```diff
@@ -48,14 +48,14 @@ List<PaymentRequestDocument> getPaymentRequestsToExtract(String campusCode, Inte
 
     /**
      * @param onlySpecialPayments true only include special payments, False include all
-     * @param chartCode           if not null, limit results to a single chart
+     * @param campusCode           if not null, limit results to a single campus
      * @return Collection of payment requests that need to be extracted to PDP.
      */
     List<PaymentRequestDocument> getPaymentRequestsToExtract(boolean onlySpecialPayments, String chartCode,
```

Comment:
I think the second parameter needs to be renamed to "campusCode".

Reply:
updated